meta-balena-common/recipes-devtools/go/go-runtime.inc

```diff
@@ -45,6 +50,8 @@ do_install() {
 	rm -rf ${D}${libdir}/go/pkg/tool
 	rm -rf ${D}${libdir}/go/pkg/obj
 	rm -rf ${D}${libdir}/go/pkg/bootstrap
+	# the cmd directory is built for the native arch so if BUILD == TARGET
+	rm -rf ${D}${libdir}/go/pkg/${BUILD_GOTUPLE}/cmd
```

Comment:
Some of these changes come from upstream commits (this one is 4f942c272d4417b5b719df25b80a6a6b54669a73) - please cherry pick those commit individually as git can then perform rebases over upstream simplifying maintenance.